doc/QuickStart.md

```diff
@@ -50,49 +50,62 @@ To make sure that your system is correctly setup, we are going to run a basic QC
-Please note that anyone running o2-qc-run-basic publishes the same object and you might see the one published by someone else. 
+Please note that anyone running o2-qc-run-basic publishes the object under the same name and you might see the one published by someone else. 
 
 ![alt text](images/basic-qcg1.png)
 ![alt text](images/basic-qcg2.png)
 
-TODO add a link to the user documentation of the QCG
+TODO add a link to the user documentation of the QCG when it is written.
 
 __Configuration file__
 
-The devices are configured in the config file named `basic.json`. It is installed in `$QUALITYCONTROL_ROOT/etc`. Each time you rebuild the code, `$QUALITYCONTROL_ROOT/etc/basic.json` is overwritten by the file in the source directory (`~/alice/QualityControl/Framework/basic.json`).
+In the example above, the devices are configured in the config file named `basic.json`. It is installed in `$QUALITYCONTROL_ROOT/etc`. Each time you rebuild the code, `$QUALITYCONTROL_ROOT/etc/basic.json` is overwritten by the file in the source directory (`~/alice/QualityControl/Framework/basic.json`).
+
+The configuration for the QC is made of many parameters described in an [advanced section of the documentation](https://github.com/AliceO2Group/QualityControl/blob/master/doc/Advanced.md#configuration-files-details). For now we can just see below the definition of a task. `moduleName` and `className` specify respectively the library and the class to load and instantiate to do the actual job of the task. 
+```json
+(...)
+"tasks": {
+  "QcTask": {
+    "active": "true",
+    "className": "o2::quality_control_modules::skeleton::SkeletonTask",
+    "moduleName": "QcSkeleton",
+    "cycleDurationSeconds": "10",
+(...)
+```
+Try and change the name of the task by replace `QcTask` by a name of your choice. Relaunch the workflows. You should now see the object published under a different directory in the QCG.
 
 ### Readout chain
 
-In this second example, we are going to use the Readout as data source.
+In this second example, we are going to use the Readout as our data source.
 
 ![alt text](images/readout-schema.png)
 
-This workflow is a bit different from the basic one. The _Readout_ is not a device and thus we have to have a _proxy_ to get data from it. This is the extra box going to the _Dispatcher_, which then injects data to the task. This is handled in the _Readout_ if you enable the corresponding configuration flag.
-
-TODO make the qc task use the daq code
+This workflow is a bit different from the basic one. The _Readout_ is not a DPL device (it is a FairMQ device though) and thus we have to have a _proxy_ to get data from it. This is the extra box going to the _Data Sampling_, which then injects data to the task. This is handled in the _Readout_ as long as you enable the corresponding configuration flag.
```

Comment:
Readout isn't in fact a FairMQ device - it just creates FairMQChannels.

Reply:
ok